src/rgw/rgw_admin.cc

```diff
@@ -931,7 +931,7 @@ static SimpleCmd::Commands all_cmds = {
   { "datalog autotrim", OPT::DATALOG_AUTOTRIM },
   { "datalog trim", OPT::DATALOG_TRIM },
   { "realm create", OPT::REALM_CREATE },
-  { "realm delete", OPT::REALM_DELETE },
+  { "realm rm", OPT::REALM_DELETE },
```

Comment:
if we should keep an entry of 
```c++
{ "realm delete", OPT::REALM_DELETE },
```
please just leave a comment. and i will do so.